src/masoniteorm/query/QueryBuilder.py

```diff
@@ -1546,6 +1546,9 @@ def when(self, conditional, callback):
             callback(self)
         return self
 
+    def truncate(self, foreign_keys=False):
+        return self.get_schema_builder().truncate(self._table, foreign_keys=False)
```

Comment:
maybe I should do instead ?
```python
self.grammar.truncate_table(self._table)
```

Reply:
Yeah exactly. We have to duplicate that sql statement in each query grammar class

Reply:
I don't think it will be a good idea to bind the query and schema classes together so we'll just have to duplicate sql in order to make it less rigid